inspirehep/base/templates/format/record/Inspire_Default_HTML_brief_macros.tpl

```diff
@@ -18,8 +18,7 @@
 #}
 
 {% macro render_doi() %}
-  {% if record.get('dois')| is_list() %}
-    {% set filtered_doi = record.get('dois.value')| remove_duplicates() %}
+    {% set filtered_doi = record.get('dois.value')|tranform_nested_list()|remove_duplicates() %}
```

Comment:
You are right in the dojson we do not have this case. The issue comes when we do this think:
{{ record.get('dois.value') }}
Actually this transforms 
<img width="909" alt="screen shot 2016-01-27 at 15 23 28" src="https://cloud.githubusercontent.com/assets/7301890/12616156/f457e90a-c509-11e5-94ea-f534e8b07509.png">
to this : 
<img width="797" alt="screen shot 2016-01-27 at 15 24 14" src="https://cloud.githubusercontent.com/assets/7301890/12616187/0fedec6e-c50a-11e5-8660-2bd18eefea94.png">
Do you think that it would be better if we avoid this style of getting the value?